arches_vue_utils/src/arches_vue_utils/types.ts

```diff
@@ -6,3 +6,19 @@ export interface Language {
     name: string;
     scope: string;
 }
+
+export interface Label {
+    value: string;
+    language_id: string;
+    valuetype_id: string;
+}
+
+export interface WithLabels {
+    labels: Label[];
+}
+
+export interface WithValues {
+    values: Label[];
+}
+
+export type Labellable = WithLabels | WithValues;
```

Comment:
not blocking but if these types end up getting used elsewhere I'd say we update them to be a touch more descriptive eg (`ItemWithLabels`)

but not too worried about it now 👍

Reply:
I hear you. I don't think they'll get used elsewhere. What's been maddeningly difficult here is that "item" has a technical meaning in the controlled list manager, so I was hoping to avoid that.